File: web-service/api-project/src/main/java/io/ehdev/conrad/service/api/service/model/project/RepoDefinitionsDetails.java
package io.ehdev.conrad.service.api.service.model.project;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonPropertyDescription;
import org.springframework.hateoas.ResourceSupport;

public class RepoDefinitionsDetails extends ResourceSupport {
    @JsonPropertyDescription("Name of the project.")
    @JsonProperty("name")
    String name;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: api/rest-model/src/main/kotlin/io/ehdev/conrad/model/project/RepoDefinitionsDetails.kt
package io.ehdev.conrad.model.project

import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.annotation.JsonPropertyDescription
import io.ehdev.conrad.model.DefaultResourceSupport

class RepoDefinitionsDetails(
    @JsonPropertyDescription("Name of the project.") @JsonProperty("name") val name: String) : DefaultResourceSupport() {
}
